app/views/criteria/new.js.erb

```diff
@@ -6,13 +6,13 @@ document.getElementById('new_criterion_prompt').focus();
 
 $(document).ready(function() {
   $('input[name=criterion_type]:radio').change(function () {
-
+    var current = document.getElementById('max_mark_prompt')
```

Comment:
Use `const` instead of `var` here. (See https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75 for further reading.)

Also, choose a better variable name than `current`.

And finally, add a semicolon to the end of the line.